tianshou/trainer/offline.py

```diff
@@ -68,11 +68,12 @@ def offline_trainer(
     stat: Dict[str, MovAvg] = defaultdict(MovAvg)
     start_time = time.time()
     test_collector.reset_stat()
-
+    test_episode(policy, test_collector, test_fn, 0,
+                 episode_per_test, writer, gradient_step, reward_metric)
```

Comment:
why not set best_reward to this result for initialization?

Reply:
done